src/VirtoCommerce.CustomerExportImportModule.Web/Scripts/blades/import-processing.tpl.html

```diff
@@ -0,0 +1,55 @@
+<div class="blade-content customer-export-import">
+    <div class="blade-inner">
+        <div class="inner-block">
+            <div class="form">
+
+                <div class="sub-t import-result-title">{{blade.notification.description}}</div>
+
+                <div class="block progress-wrap" ng-if="blade.notification.created || blade.notification.finished">
```

Comment:
Here and below: why these `if` exists? It hide values if they have 0. Check as `!!x` instead if you need to hide them when undefined